{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -10,54 +10,56 @@ readme = "README.md"
 authors = ["{{ cookiecutter.project_domain }}"]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.12"
 
-django = "^3.2"
-django-hurricane = "^0.8.8"
-django-split-settings = "^1.0"
-django-csp = "^3.7"
+django = "^5.0"
```

Comment:
We usually use the LTS version of Django, which currently is 4.2.x. Did you pin it to 5.0 intentionally?

Reply:
Ah makes sense, will change it to 4.2 👍